src/cascadia/TerminalApp/CommandPalette.xaml

```diff
@@ -0,0 +1,171 @@
+                    <SolidColorBrush x:Key="TextControlButtonForegroundPressed" Color="#FFFFFF"/>
+                    <SolidColorBrush x:Key="TextControlButtonBackgroundPressed" Color="#FF4343"/>
+
+                </ResourceDictionary>
+                <ResourceDictionary x:Key="HighContrast">
+                    <Style x:Key="CommandPaletteBackground" TargetType="Grid">
+                        <Setter Property="Background" Value="{ThemeResource SystemColorWindowColor}" />
+                    </Style>
+                </ResourceDictionary>
+            </ResourceDictionary.ThemeDictionaries>
+        </ResourceDictionary>
+
+    </Grid.Resources>
+
+    <Grid.ColumnDefinitions>
+        <ColumnDefinition Width="*" />
+        <ColumnDefinition Width="8*" />
+        <ColumnDefinition Width="*" />
+    </Grid.ColumnDefinitions>
+
+    <Grid.RowDefinitions>
+        <RowDefinition Height="9*"/>
+        <RowDefinition Height="*"/>
+    </Grid.RowDefinitions>
+
+    <Grid
+        x:Name="ShadowBackdrop"
+        Background="Transparent"
+        Grid.Column="1"
+        Grid.Row="0"
+        HorizontalAlignment="Stretch"
+        VerticalAlignment="Stretch">
+        </Grid>
+
+    <Grid
+        x:Name="Backdrop"
+        Style="{ThemeResource CommandPaletteBackground}"
+        CornerRadius="{ThemeResource ControlCornerRadius}"
+        Margin="8"
+        Grid.Column="1"
+        Grid.Row="0"
+        MaxWidth="400"
+        MaxHeight="400"
+        Windows10version1903:Shadow="{StaticResource CommandPaletteShadow}"
+        HorizontalAlignment="Stretch"
+        VerticalAlignment="Stretch">
+
+        <Grid.RowDefinitions>
+            <RowDefinition Height="Auto"/>
+            <RowDefinition Height="*"/>
+        </Grid.RowDefinitions>
+
+        <TextBox
+            Grid.Row="0"
+            x:Uid="CommandPalette_SearchBox"
+            x:Name="_SearchBox"
+            Margin="8"
+            TextChanged="_filterTextChanged"
+            KeyDown="_keyDownHandler"
+            Text="">
+        </TextBox>
+
+        <ListView
+            Grid.Row="1"
+            x:Name="_FilteredActionsView"
+            HorizontalAlignment="Stretch"
+            VerticalAlignment="Stretch"
+            SelectionMode="Single"
+            CanReorderItems="False"
+            AllowDrop="False"
+            ItemsSource="{x:Bind FilteredActions}">
+
+            <ItemsControl.ItemTemplate >
+                <DataTemplate x:DataType="local:Command">
+
+                    <!-- This HorizontalContentAlignment="Stretch" is important
+                    to make sure it takes the entire width of the line -->
+                    <ListViewItem HorizontalContentAlignment="Stretch"
+                                  AutomationProperties.Name="{x:Bind Name, Mode=OneWay}"
+                                  AutomationProperties.AcceleratorKey="{x:Bind KeyChordText, Mode=OneWay}">
```

Comment:
_scary!_ unless this is intentionally a free-form text field.

Reply:
This is what @carlos-zamora and I decided on for accesibility, I believe he was going to confirm that it worked as expected